docs/source/index.rst

```diff
@@ -48,7 +48,12 @@ on command-line. The following configuration is useful to setup.
 Basic Configuration
 --------------------
 
-Flytectl configuration
+Flytectl configuration. 
```

Comment:
doesn't seem like this line is necessary

Reply:
yeah. I have removed this now